### PR TITLE
Add Async support docs

### PR DESCRIPTION
**Pull request by marinhero**

### CODE REVIEW

File: source/includes/_install_js_intro.md [hunk 1]
```diff
@@ -66,3 +68,24 @@ completion page.
 #### Google Tag Manager:
 You may use Google Tag Manager to install and manage the Wootric
 snippet.
+
+### Async Support:
+We support HTML5 async attribute to load our snippet.
+The wootricSettings object is built in the same way as above. The only difference
+rests in how we call the WootricSurvey.
```
Comment (hachibu) [inline source/includes/_install_js_intro.md]: Maybe: `how we initialize WootricSurvey.`

File: source/includes/_install_js_intro.md [hunk 2]
```diff
@@ -18,6 +18,8 @@ Once you are signed up on the Wootric homepage, you will be taken directly to an
 page. If you’re a returning visitor, sign in at [wootric.com](https://www.wootric.com/) and click on the “Settings" button near the top right of the page. Navigate to the [Javascript Setup Guide](https://app.wootric.com/install) and you will see a code snippet with a unique
 account_token for you to install.
 
+For HTML5 async support read [here](#async-support:)
```
Comment (hachibu) [inline source/includes/_install_js_intro.md]: Remove the extra colon so this works.

File: source/includes/_install_js_intro.md [hunk 3]
```diff
@@ -66,3 +68,24 @@ completion page.
...
 You may use Google Tag Manager to install and manage the Wootric
 snippet.
+
+### Async Support:
+We support HTML5 async attribute to load our snippet.
+The wootricSettings object is built in the same way as above. The only difference
+rests in how we call the WootricSurvey.
+Take a close look at the onload attribute, it will guarantee that your service is called when the script has loaded.
+
+Example:
```
Comment (hachibu) [inline source/includes/_install_js_intro.md]: Remove this text. It's placed way to the side.

**Comment by diegoserranoa** [comment]
screen shot 2017-12-14 at 1 23 13 pm

Maybe you can try putting it like this so it aligns better?
```
### Async Support:
``` js
<!-- begin Wootric code -->
<script type="text/javascript">
   wootric_survey_immediately = true;
   window.wootricSettings = {
      email: 'customer@example.com',
      created_at: 1234567890,
      account_token: 'NPS-XXXXXXXX'
  };
</script>
<script type="text/javascript" src="https://disutgh7q0ncc.cloudfront.net/beacon.js" async onload="WootricSurvey.run(wootricSettings)"></script>
``
We support HTML5 async attribute to load our snippet.
The wootricSettings object is built in the same way as above. The only difference
rests in how we initialize the WootricSurvey.
Take a close look at the onload attribute, it will guarantee that your service is called when the script has loaded.
```
PS: I removed the last \`  after `</script>`

**Comment by marinhero** [comment]
Done @diegoserranoa !

**Comment by diegoserranoa** [comment]
screen shot 2017-12-14 at 2 39 31 pm

:shipit:

**Review by diegoserranoa — APPROVED** [review]

**Pull request merged**